### ISSUE TITLE
go-pry not compatible with various imported packages?

### ISSUE THREAD

**Issue by adamgeorgeson** [d4l3k/go-pry#7]
I was looking to use go-pry, as a big fan of pry in Ruby.

I'm quite new to Go, but I have found that if I try to use go-pry on anything that's using more than the fmt package, I get issues.

An example is if I have `test.go` as below. Can use go-pry without importing the os package, but not with. I've come across other packages such as `path/filepath` that give similar errors.

```
package main

import "github.com/d4l3k/go-pry/pry"
import "os"

func main() {
  pry.Pry()
  os.Setenv("foo", "bar")
}
```

```
~/dev/go/src/github.com/adamgeorgeson/test$ go-pry run test.go
# command-line-arguments
./test.go:7: duplicate key "Pipe" in map literal
./test.go:7: duplicate key "Interrupt" in map literal
./test.go:7: duplicate key "Kill" in map literal
./test.go:7: duplicate key "Remove" in map literal
./test.go:7: duplicate key "TempDir" in map literal
./test.go:7: duplicate key "NewFile" in map literal
./test.go:7: duplicate key "DevNull" in map literal
./test.go:7: duplicate key "Symlink" in map literal
./test.go:7: duplicate key "OpenFile" in map literal
./test.go:7: too many errors
Reverting files
```

Cheers,
Adam.

**Comment by d4l3k** [comment]
Sorry for taking so long to get around to this. I was interning at a fairly large corporation and it made it rather hard for me to contribute to open source code.

I took a look at those packages and it's caused by duplicated exports due to the "// +build ..." rules.
I've fixed this and it should be much better.

I'm hoping to write some better automated tests for all the std library packages in the near future.
